MATLAB/Demos/demo_for_FDK_Half_based_on_d14_Offsets.m

```diff
@@ -0,0 +1,60 @@
+%% DEMO 14:  Playing with offsets
+%
+%
+% In this demo we show how to change offsets to either the iamge or the
+% detector, and the flexibility of it.
+% 
+% 
+%--------------------------------------------------------------------------
+%--------------------------------------------------------------------------
+% This file is part of the TIGRE Toolbox
+% 
+% Copyright (c) 2015, University of Bath and 
+%                     CERN-European Organization for Nuclear Research
+%                     All rights reserved.
+%
+% License:            Open Source under BSD. 
+%                     See the full license at
+%                     https://github.com/CERN/TIGRE/blob/master/LICENSE
+%
+% Contact:            tigre.toolbox@gmail.com
+% Codes:              https://github.com/CERN/TIGRE/
+% Coded by:           Ander Biguri 
+%--------------------------------------------------------------------------
+%% Initialize
+clear;
+close all;
+
+%% Define Geometry
+geo=defaultGeometry('nVoxel',[256;256;256]);                     
+
+% Offsets
+%% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+% Lets try simnple offset: The detector gets completelly displaced
+geo.offOrigin = [0;0;0];                     % Offset of image from origin   (mm)              
+geo.offDetector=[180; 0];                     % Offset of Detector            (mm)

```

Comment:
This is described as half offset all along this PR, but in theory it should work for any type of offset, right? Not only half offsets. Is the code in `FDK_half()` hardcoded somehow to onyl work with half offsets? If so, could we change it to allow for any offset?

Reply:
_half_ here or in some papers/books doesn't mean the beam angle has be to exactly 0.5* fan angle. In the code, _geo.offDetector=[**X**; 0];_   the X can be any value. The displacement is calculated and compensated in **_zeropadding_** and **_preweighting_**.